app/client/craco.common.config.js

```diff
@@ -20,6 +20,11 @@ module.exports = {
   eslint: {
     enable: false,
   },
+  typescript: {
+    enableTypeChecking:
+      process.env.ENABLE_TYPE_CHECKING === "true" ||
+      process.env.ENABLE_TYPE_CHECKING === undefined,
```

Comment:
Minor. Can be simplified to just one condition?

```suggestion
      process.env.ENABLE_TYPE_CHECKING !== "false",
```

This way, even if it is set to empty string, it'll still do the right thing.

Reply:
@sharat87 Thanks, it's much better this way.